app/services/unschedule_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class UnscheduleService
+  attr_reader :edition
+
+  def initialize(edition)
+    @edition = edition
+  end
+
+  def unschedule(user:)
+    set_edition_status(user)
+    destroy_publishing_intent
+  end
+
+private
+
+  def set_edition_status(user)
+    updater = Versioning::RevisionUpdater.new(edition.revision, user)
+    edition.assign_revision(updater.next_revision, user).save!
```

Comment:
I think this code isn't needed anymore, I assume it's from when we were setting the date to nil?

Reply:
Yes, good catch thank you.